test/index.js

```diff
@@ -0,0 +1,18 @@
+var request = require('request'),
+    msg = process.env.TRAVIS_COMMIT_MSG;
```

Comment:
Makes sense. Should we update the filename/npm script name to make it clear that this test only runs in the context of Travis, then?